libraries/lib-time-and-pitch/TimeAndPitchExperimentalSettings.cpp

```diff
@@ -9,22 +9,36 @@
 
 **********************************************************************/
 #include "TimeAndPitchExperimentalSettings.h"
+#include <cstdlib>
 #include <fstream>
+#include <string>
+
+// #define TIME_AND_PITCH_TUNING
 
 namespace
 {
-const std::string logDir { CMAKE_CURRENT_SOURCE_DIR };
+
+std::string GetLogDir()
+{
+   const char* appDataDir = getenv("APPDATA");
```

Comment:
This code only works on Windows

Reply:
That's all right, I only need it there, and if someone needs it elsewhere, she can generalize it.